partcad/src/partcad/test/cam.py

```
@@ -0,0 +1,117 @@
+class CamTest(Test):
+    def __init__(self) -> None:
+        super().__init__("cam")
+
+    async def test(self, tests_to_run: list[Test], ctx, shape, test_ctx: dict = {}) -> bool:
+        is_part = isinstance(shape, Part)
+        is_assembly = isinstance(shape, Assembly)
+        if not is_part and not is_assembly:
+            self.debug(shape, "Not applicable")
+            return True
+
+        if not shape.is_manufacturable and not "force_manufacturing" in test_ctx:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)